client/layout/style.scss

```diff
@@ -40,6 +40,12 @@
 		margin: 0;
 	}
 
+	.is-section-theme & {
+		.sites.main {
+			padding-top: 79px;
+		}
+	}
```

Comment:
This style should also be somewhere in `my-sites/theme`. We should be removing section-specific styles from the common layout stylesheet instead of adding new ones.